portfolio/src/main/webapp/script.js

```diff
@@ -12,24 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Adds random fun message to the page.
-function addFunMessage() {
-    const funMessages =
-        ['Henlo', 'Nalonalotulonie', 'I eat rocks.'];
-
-    // Randomize the fun message.
-    const message = funMessages[Math.floor(Math.random() * funMessages.length)];
-    
+// Adds comment to the page.
+function addComment() {
+    // Create comment.
+    const comment;
     // Add it to the page.
-    const messageContainer = document.getElementById('message-container');
-    messageContainer.innerText = message;
+    const commentContainer = document.getElementById('comment-container');
+    commentContainer.innerText = comment;
```

Comment:
It looks like `comment` has not been set to any value -- does it need to be set before being written to `commentContainer`?

Reply:
Sorry, I should've commented this function out. It has some changes that I didn't go through with, but left accidentally.

Reply:
Ok understood, thanks!